sunpy/instr/tests/test_fermi.py

```diff
@@ -50,3 +53,8 @@ def test_detector_angles():
     assert_almost_equal(det2['n7'].value, 127.35783, decimal=1)
     assert_almost_equal(det2['n8'].value, 122.98894, decimal=1)
     assert_almost_equal(det2['n9'].value, 126.95987, decimal=1)
+
+
+def test_met_to_utc():
+    time = fermi.met_to_utc(500000000)
```

Comment:
@aringlis @wafels We discovered by doing this that the result when converting Fermi MET to UTC is 2s different when using astropy Time than the previous datetime implementation. We talked through this and we think that it's because the previous implementation was not taking into account the two UTC leap seconds since the start of Fermi MET. Do either of you have any ideas if we could validate this somehow?

Reply:
@Cadair hmm, interesting. A quick test is trying some different input time values, to see if the same 2s discrepancy is always there, e.g. try timestamps before and after the leap second.

I can also look into what the Fermi team itself does with their MET values.

Reply:
See Issue #2710